server/helpers/cronjobs/index.js

```diff
@@ -4,13 +4,17 @@ const giveBadges = require("./giveBadges");
 const sendScheduledReminders = require("./sendScheduledReminders");
 const sendScheduledPaymentReminders = require("./sendScheduledPaymentReminders");
 const releaseExpiredCouponsValue = require("./releaseExpiredCouponsValue");
-
+const cancelOldBookings = require("./cancelOldBookings");
 
 const cronJobs = async (Sentry) => {
   cron.schedule("1 1 1 * * *", async () => {
     await releaseExpiredCouponsValue(Sentry);
   });
 
+  cron.schedule("* * 0 * * *", async () => {
```

Comment:
As Ramy has told me this will work every second of every minute in the hour 0 of every month of every year,
So I think the correct way we want it to work only at the first second on the first minute of the hour 0 every day of every month of every year :P
```suggestion
  cron.schedule("1 1 0 * * *", async () => {
```

Reply:
@ashatat 
Correct, I haven't noticed that, but, it'll be wrong to set it to `1 1 0` that will mean every day at 12:01:01 AM which means bookings that have the start date to the same day will be removed `(12:00:00 < 12:01:01)`  so it should be `0 0 0 ...`

Reply:
but at `12:00:00` won't be removed? though it is considered the same day?